Yona/Yona/Welcome/WelcomeViewController.swift

```diff
@@ -83,30 +83,41 @@ class WelcomeViewController: BaseViewController {
     
     fileprivate func processEnvironmentURL(newEnvironmentURL:String, currentEnvironmentURL:String) {
         EnvironmentManager.updateEnvironment(withURLString: newEnvironmentURL)
-        self.didEditTextField = false
-        if currentEnvironmentURL != newEnvironmentURL{
+        if (currentEnvironmentURL != newEnvironmentURL) {
             ActivitiesRequestManager.sharedInstance.getActivityDefaultCategories(onCompletion: { (success, message, code, activitiesReturned, error) in
-                if success { EnvironmentManager.updateEnvironment(withURLString: newEnvironmentURL)
-                }else{ EnvironmentManager.updateEnvironment(withURLString: currentEnvironmentURL)
+                if success {
+                    EnvironmentManager.updateEnvironment(withURLString: newEnvironmentURL)
+                    self.showEnvironmentChangedAlert(message: NSLocalizedString("environment-success", comment: "") + newEnvironmentURL)
+                } else {
+                    EnvironmentManager.updateEnvironment(withURLString: currentEnvironmentURL)
+                    self.showEnvironmentChangedAlert(message: NSLocalizedString("environment-failure", comment: "") + currentEnvironmentURL)
                 }
             })
-        }else{
-            if (self.didEditTextField && currentEnvironmentURL != newEnvironmentURL){
-                self.showSameEnvironmentURLAlert()
-            }
+        } else if (self.didEditTextField) {
+            self.showSameEnvironmentURLAlert()
+        } else {
```

Comment:
Please remove this and the next one, as there is no reason to have an else statement that explicitly does nothing.

Reply:
Removed useless else statement